Source/Softswitch/inc/poets_msg.h

```diff
@@ -3,69 +3,113 @@
 
 #include <cstddef>
 #include <cstdint>
+#include "config.h"           // Tinsel config
 
-// poets_hardware.h is a platform-specific configuration file
-// generated by the Orchestrator, and which must include at least
-// P_MSG_MAX_SIZE, LOG_BOARDS_PER_BOX, LOG_CORES_PER_BOARD,
-// and LOG_THREADS_PER_CORE
 #include "poets_hardware.h"
+#include "SoftwareAddressDefs.h"
 
-#define DEST_BROADCAST 0xFFFFFFFF
 
-#define MAX_P_SUP_MSG_BYTES 256
-#define LOG_DEVICES_PER_THREAD 10
-#define P_DEVICE_OS 0
-#define P_THREAD_OS (LOG_DEVICES_PER_THREAD)
-#define P_CORE_OS (LOG_THREADS_PER_CORE+P_THREAD_OS)
-#define P_BOARD_OS (LOG_CORES_PER_BOARD+P_CORE_OS)
-#define P_BOX_OS (LOG_BOARDS_PER_BOX+P_BOARD_OS)
-#define P_SUP_OS 31
-#define P_SUP_MASK (0x1 << P_SUP_OS)
+//------------------------------------------------------------------------------
```

Comment:
I "feel" like these should be in their own header.

Reply:
Quite probably. I put them here when I was messing about with the Instrumentation concept as it was a common source file used by both TMoth and the Softswitch.

Suggestions on a more appropriate place to put it? softswitch_defs.h?

Reply:
`softswitch_instrumentation_defs.h`?

Reply:
could do, though is there anything else that is likely to be shared between mothership and softswitch? if so, `softswitch_defs.h` might be more appealing?